lib/ansible/modules/network/panos/_panos_security_policy.py

```diff
@@ -36,9 +36,11 @@
 author: "Ivan Bojer (@ivanbojer)"
 version_added: "2.3"
 deprecated:
-  removed_in: "2.8"
-  why: Renamed to M(panos_security_rule) in order to align with API calls and UI object references, which also has extra support for PanDevice SDK.
-  alternative: Use M(panos_security_rule) instead.
+    alternative:
```

Comment:
`alternative: Use M(panos_security_rule) instead.`

`why: This module depended on outdated and old SDK.`